examples/react-flow/src/App.tsx

```
@@ -0,0 +1,106 @@
+  );
+
+  const onEdgesChange = useCallback(
+    (changes: Array<EdgeChange>) => {
+      update(() => {
+        for (const c of changes) {
+          switch (c.type) {
+            case 'add':
+              root.edges.push(c.item);
+              break;
+            case 'replace':
+              const replace = root.edges.findIndex((e) => e.id === c.id);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)